src/view/concept-card.inc.twig

```diff
@@ -1,4 +1,4 @@
-<div class="col-md-8" id="main-content">
+<div class="col-md-8 ps-3" id="main-content">
```

Comment:
Please adjust the gutter instead here: https://github.com/NatLibFi/Skosmos/blame/53aa787e6269859f5cb8739ea327ced0a2322453/resource/css/skosmos.css#L359